src/components/Modal/index.tsx

```diff
@@ -61,8 +51,6 @@ class BaseModal extends React.Component<ThemedModalProps, any> {
 		if (!BaseModal.mountedCount) {
 			document.body.classList.remove(bodyNoOverflowClass);
 		}
-
-		window.removeEventListener('keydown', this.handleKeyDown);
```

Comment:
Any reason we don't need this anymore? I think this might have been done for nested modals, not sure..

Reply:
this was a mistake, I was trying to move the entire `keydown` event on `ActionsButtons`, but then I realized that we don't want it, and I forgot to add the `removeEventListener` back. :pray: